app/components/team/Index.js

```diff
@@ -93,16 +93,16 @@ class TeamIndex extends React.PureComponent {
         <div>
           <Panel>
             <Panel.Section className="max-width-3">
-              <p>Teams allows you to group people, pipelines and permissions together.</p>
+              <p>Teams allows you to assign permissions to different groups of users and pipelines. You can create secret teams which are only visible to account admins and team members, or public teams that are visible to the your entire Buildkite organization.</p>
```

Comment:
What do you think about this for the first sentence?
> Creating teams inside your organisation allows you to assign permissions to pipelines and groups of users.

I found that having the start the same as the top section of the page made me skip this paragraph altogether.